docs/src/pages/components/tree-view/FileSystemNavigator.js

```diff
@@ -1,27 +1,16 @@
 import * as React from 'react';
-import { makeStyles } from '@material-ui/core/styles';
 import TreeView from '@material-ui/lab/TreeView';
 import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
 import ChevronRightIcon from '@material-ui/icons/ChevronRight';
 import TreeItem from '@material-ui/lab/TreeItem';
 
-const useStyles = makeStyles({
-  root: {
-    height: 240,
-    flexGrow: 1,
-    maxWidth: 400,
-  },
-});
-
 export default function FileSystemNavigator() {
```

Comment:
If we want to look into inline previews at the same time (not sure we want), this could help:

```diff
diff --git a/docs/src/pages/components/tree-view/FileSystemNavigator.tsx b/docs/src/pages/components/tree-view/FileSystemNavigator.tsx
index d4102e3ec3..90023450b9 100644
--- a/docs/src/pages/components/tree-view/FileSystemNavigator.tsx
+++ b/docs/src/pages/components/tree-view/FileSystemNavigator.tsx
@@ -14,16 +14,11 @@ export default function FileSystemNavigator() {
     >
       <TreeItem nodeId="1" label="Applications">
         <TreeItem nodeId="2" label="Calendar" />
-        <TreeItem nodeId="3" label="Chrome" />
-        <TreeItem nodeId="4" label="Webstorm" />
       </TreeItem>
       <TreeItem nodeId="5" label="Documents">
         <TreeItem nodeId="10" label="OSS" />
         <TreeItem nodeId="6" label="Material-UI">
-          <TreeItem nodeId="7" label="src">
-            <TreeItem nodeId="8" label="index.js" />
-            <TreeItem nodeId="9" label="tree-view.js" />
-          </TreeItem>
+          <TreeItem nodeId="8" label="index.js" />
         </TreeItem>
       </TreeItem>
     </TreeView>
```

<img width="868" alt="Screenshot 2021-05-05 at 18 07 41" src="https://user-images.githubusercontent.com/3165635/117173272-e57c6300-adcc-11eb-80c0-b7ba1f3ffed7.png">